src/tiledb/cloud/vcf/ingestion.py

```diff
@@ -0,0 +1,977 @@
+import enum
+import inspect
+import logging
+import subprocess
+from collections import defaultdict
+from math import ceil
+from typing import Any, Mapping, Optional, Sequence, Tuple, Union
+
+import numpy as np
+import tiledbvcf
+
+import tiledb.cloud
+from tiledb.cloud.utilities import Profiler
+from tiledb.cloud.utilities import create_log_array
+from tiledb.cloud.utilities import get_logger
+from tiledb.cloud.utilities import run_dag
+from tiledb.cloud.utilities import set_aws_context
+from tiledb.cloud.utilities import write_log_event
+
+LOCAL_INGEST = False
+VERBOSE = False
+TRACE = False
+
+
+def local_ingest():
```

Comment:
I don't entirely understand the intent of these functions. Could a user not simply set the module-level variable themselves?

If you want to keep them, it would be helpful to give them a name that reflects that they are taking an action, maybe `set_local_ingest`.

Reply:
These functions can go away. They were added with some other code to test the code on Cloud without putting everything in one file, and they probable weren't necessary then either.